R/tmb-methods.R

```diff
@@ -429,3 +429,152 @@ h_residuals_response <- function(object) {
   assert_class(object, "mmrm_tmb")
   component(object, "y_vector") - unname(fitted(object))
 }
+
+
+#' @describeIn mmrm_tmb_methods Simulates from an mmrm model object
+#' @importFrom stats simulate
```

Comment:
we have `package-mmrm.R` to hold all imported function

Reply:
All the other `describeIn` tags point to `mmrm_tmb_methods`. @clarkliming are you thinking we should change this?

Reply:
oh true for these methods we have additional `importFrom`. then consistency is fine